charts/pact-broker/values.yaml

```diff
@@ -35,8 +35,11 @@ broker:
   # -- Additional annotations that can be added to the Broker deployment
   annotations: {}
 
-  # -- Number of Pact Broker replicas to deploy
-  replicaCount: 1
+  # -- (int) Number of Pact Broker replicas to deploy
+  replicaCount:
```

Comment:
Not should why we are removing the default value here of 1?

Reply:
Typo, apologies, was going fast last night